### DISCUSSION TITLE
GitHub App is removed but it is still able to run!

### DISCUSSION THREAD

**Discussion by billytronics** [community/community#35657] [Apps, API and Webhooks]
An installed GitHub app that accesses my PRs and run a scan of the code was manually removed and after triggering a new PR, the app is still running its tests. This is even after revoking its access and then removing the app entirely from my account (by uninstalling the app via: Settings > Applications > uninstall).
I'm assuming access was granted one time and there is now no way to remove such access? Surely this is a security issue?

Thanks.

**Comment by jfullstackdev** [comment]
![image](https://user-images.githubusercontent.com/47092464/194876326-d1b273b3-f1b4-4b99-af8b-33253fa8d184.png)

https://github.com/settings/installations

did you click all the tabs ? 

how about here, 

![image](https://user-images.githubusercontent.com/47092464/194876543-fd88fcbc-8706-4d96-b711-789294580367.png)

https://github.com/settings/apps

**Reply by billytronics** [reply]
Thanks for the suggestions - I reviewed those places and it is no longer there. Just created a new PR and can confirm that the offending app is still triggered, in the checks.
The PR is picking up previous commits that I had not merged yet, commits that were done before I removed the GitHub App. Is the authorized access referring to when the commits were done? Otherwise, it seems like GitHub did not remove the access when the app was removed.

**Reply by jfullstackdev** [reply]
when did you delete that ? that might take time to take effect

**Reply by billytronics** [reply]
I deleted it earlier today. Let me come back tomorrow and try again.

**Reply by ettaboyle** [reply]
@billy-yung Thanks for posting here - and @jdevstatic thanks for your response
@billy-yung Can you let us know if this was resolved?

**Reply by billytronics** [reply]
@ettaboyle - thanks for checking on this. I think I found the problem now - if I create a new PR on the same commit, it will simply present the old results from the previous checks (by the app). I just tested by raising a new PR off a new commit and see that the check by the removed app is gone now.
I'll mark this as resolved, thanks all.
